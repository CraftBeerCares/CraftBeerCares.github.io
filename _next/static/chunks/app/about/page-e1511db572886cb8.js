(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[301,327,38,406],{8840:function(e,t,n){Promise.resolve().then(n.t.bind(n,3222,23)),Promise.resolve().then(n.t.bind(n,6685,23)),Promise.resolve().then(n.t.bind(n,6993,23)),Promise.resolve().then(n.t.bind(n,860,23))},6993:function(e){e.exports={footer:"footer_footer__GoTLO"}},860:function(e){e.exports={header:"header_header__RtJAw",title:"header_title___TxJx",navLink:"header_navLink__NEYGl",active:"header_active__01PQA",burgerMenuButton:"header_burgerMenuButton__vrG4B",topNav:"header_topNav__MQVPf"}}},function(e){e.O(0,[451,971,596,744],function(){return e(e.s=8840)}),_N_E=e.O()}]);