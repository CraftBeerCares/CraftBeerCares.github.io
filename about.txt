2:I[6937,["429","static/chunks/429-479f5a9c9e758f58.js","301","static/chunks/app/about/page-89cf36820f3b5972.js"],"Header"]
3:I[1749,["429","static/chunks/429-479f5a9c9e758f58.js","301","static/chunks/app/about/page-89cf36820f3b5972.js"],"Image"]
4:I[5250,["429","static/chunks/429-479f5a9c9e758f58.js","301","static/chunks/app/about/page-89cf36820f3b5972.js"],""]
5:I[5613,[],""]
6:I[1778,[],""]
0:["EYMa613mft3IsttxDFdjl",[[["",{"children":["about",{"children":["__PAGE__",{}]}]},"$undefined","$undefined",true],["",{"children":["about",{"children":["__PAGE__",{},["$L1",["$","div",null,{"className":"pageTemplate_pageContainer__cQnRZ","children":[["$","div",null,{"className":"pageTemplate_header__9KyCG","children":["$","div",null,{"className":"pageTemplate_content__jvw6J","children":["$","$L2",null,{}]}]}],["$","div",null,{"className":"pageTemplate_scrollContainer__0V5IP","children":[["$","main",null,{"className":"__className_bc8ab8 pageTemplate_content__jvw6J","children":[["$","div",null,{"className":"articleHeader_articleHeader__PdbWz","children":[["$","div",null,{"className":"articleHeader_imageWrapper__J3d1S","children":["$","$L3",null,{"src":"/assets/003.webp","alt":"A picture of me","fill":true,"style":{"objectFit":"cover"}}]}],["$","article",null,{"className":"articleHeader_article__O845k articleHeader_articleLeft__FUd4Y","children":[["$","h1",null,{"className":"__className_b474bf","children":"The Craft Beer Cares Story"}],["$","p",null,{"children":"Craft Beer Cares was founded by Gautam Bhatnagar and Michaela Zelenanska in 2017. Inspired by other initiatives such as Cook for Syria that were happening at the time, the two took their social interests and contacts in craft beer and worked towards organising something that could raise money for great causes."}],["$","p",null,{"children":"Our first festival took place at BrewClub in Clapton, 1-2 July 2017. Since then, thanks to many friends and contacts within the craft beer industry as well as other wonderful friends, family members, volunteers, and enthusiastic attendees, we have held a grand total of 4 beer festivals within 4 years, and have raised over £50,000 for charitable organisations."}]]}]]}],["$","div",null,{"className":"articleRow_articleRow__aCKRL","children":[false,["$","article",null,{"className":"articleRow_article__Aex7B","children":[["$","p",null,{"children":"Our most recent festival, Love Beer London, took place in partnership with the London Brewers Alliance and SIBA Southeast, and took place 13-15 February 2020, only a short month before COVID-19 brought the UK into its first lockdown."}],["$","p",null,{"children":"Together, we managed to raise nearly £10,000 for The Benevolent, now called The Drinks Trust, a charity that provides support, care and assistance to people working in the drinks trade. Considering the upheaval many hospitality and drinks industry workers would find themselves in during the cycles of lockdown, furlough, reopening, and in some cases losing their jobs, we are glad to have been able to have contributed to such an incredible charity when we were able to do so."}]]}],["$","div",null,{"className":"articleRow_articleImage__g9exY","children":["$","img",null,{"src":"/assets/001.webp","alt":"alt","style":{"objectFit":"contain","width":"100%","marginLeft":"3rem"}}]}]]}],["$","div",null,{"className":"articleRow_articleRow__aCKRL","children":[["$","div",null,{"className":"articleRow_articleImage__g9exY","children":["$","img",null,{"src":"/assets/001.webp","alt":"alt","style":{"objectFit":"contain","width":"100%","marginLeft":"-3rem"}}]}],["$","article",null,{"className":"articleRow_article__Aex7B","children":[["$","p",null,{"children":"Although our current plans for our next festival event may be some months off, if not perhaps even a bit longer, we at Craft Beer Cares are keen to promote other events, organisations, fund-raisers, and programmes that are supporting charities and non-profit organisations big and small, taking place throughout the UK."}],["$","p",null,{"children":"If you'd like to receive news about charity events within the beer industry, please sign up for our newsletter. And if you have an event or an initiative you're holding for a charity or another great cause, please let us know by contacting us."}]]}],false]}],["$","div",null,{"className":"about_thanks__2cFq7","children":[["$","div",null,{"className":"quote_quoteBox__qEkdt","children":[["$","$L3",null,{"src":"/assets/quote_start.svg","alt":"Quote Start","width":24,"height":24,"className":"quote_quoteImage__hAnn1"}],["$","p",null,{"className":"quote_quote__Y6rmy","children":"The generous support from Craft Beer Cares is huge for Art Against Knives."}],["$","$L3",null,{"src":"/assets/quote_end.svg","alt":"Quote End","width":24,"height":24,"className":"quote_quoteImage__hAnn1"}],["$","p",null,{"className":"quote_author__BOC1v","children":"Katy Dawe, Art Against Knives Director"}]]}],["$","div",null,{"className":"quote_quoteBox__qEkdt","children":[["$","$L3",null,{"src":"/assets/quote_start.svg","alt":"Quote Start","width":24,"height":24,"className":"quote_quoteImage__hAnn1"}],["$","p",null,{"className":"quote_quote__Y6rmy","children":"We'd like to say a huge thank you to Craft Beer Cares for choosing to support Mind through their first beer festival. Every penny raised will fund Mind's vital work."}],["$","$L3",null,{"src":"/assets/quote_end.svg","alt":"Quote End","width":24,"height":24,"className":"quote_quoteImage__hAnn1"}],["$","p",null,{"className":"quote_author__BOC1v","children":"Karen Bolton, Community Fundraising Manager for Mind"}]]}],["$","div",null,{"className":"quote_quoteBox__qEkdt","children":[["$","$L3",null,{"src":"/assets/quote_start.svg","alt":"Quote Start","width":24,"height":24,"className":"quote_quoteImage__hAnn1"}],["$","p",null,{"className":"quote_quote__Y6rmy","children":"From November through to March, the coldest months of the year, Hackney Winter Night Shelter opens its doors to the most vulnerable in our community. Any money raised would be of enormous benefit."}],["$","$L3",null,{"src":"/assets/quote_end.svg","alt":"Quote End","width":24,"height":24,"className":"quote_quoteImage__hAnn1"}],["$","p",null,{"className":"quote_author__BOC1v","children":"Christina Ball, Hackney Winter Night Shelter"}]]}]]}]]}],["$","div",null,{"className":"pageTemplate_footer__SaaU7","children":["$","div",null,{"className":"pageTemplate_content__jvw6J","children":["$","footer",null,{"className":"$undefined","children":[["$","div",null,{"className":"footer_footerMobile__F33B9","children":["$","div",null,{"className":"footer_footerLogo__mj8G6","children":["$","$L3",null,{"src":"/assets/logo_bw.svg","alt":"Craft Beer Cares Logo","fill":true,"style":{"filter":"invert(1)"}}]}]}],["$","div",null,{"className":"footer_footerDesktop__h97Ga","children":[["$","div",null,{"className":"footer_footerColumn__vJ4D7","children":[["$","h4",null,{"className":"__className_b474bf footer_title__ua0Ts","children":"Craft Beer Cares"}],["$","nav",null,{"className":"__className_b474bf footer_footerLinks__khjBW","children":[["$","$L4","/",{"className":"footer_navLink__NX8Ib","href":"/","children":"Home"}],["$","$L4","/about",{"className":"footer_navLink__NX8Ib","href":"/about","children":"About"}],["$","$L4","/join_us",{"className":"footer_navLink__NX8Ib","href":"/join_us","children":"Join Us"}],["$","$L4","/events",{"className":"footer_navLink__NX8Ib","href":"/events","children":"Events"}],["$","$L4","/shop",{"className":"footer_navLink__NX8Ib","href":"/shop","children":"Shop"}],["$","$L4","/contact",{"className":"footer_navLink__NX8Ib","href":"/contact","children":"Contact"}]]}],["$","div",null,{"className":"footer_socialLinks__yVVge","children":[["$","a",null,{"href":"https://www.facebook.com/craftbeercares","className":"footer_socialLink__xJiBt","children":["$","$L3",null,{"src":"/assets/facebook.svg","alt":"Facebook","width":"30","height":"30"}]}],["$","a",null,{"href":"https://www.instagram.com/craftbeercares/","className":"footer_socialLink__xJiBt","children":["$","$L3",null,{"src":"/assets/instagram.svg","alt":"Instagram","width":"30","height":"30"}]}],["$","a",null,{"href":"https://twitter.com/craftbeercares","className":"footer_socialLink__xJiBt","children":["$","$L3",null,{"src":"/assets/twitter.svg","alt":"Twitter","width":"30","height":"30"}]}]]}],["$","div",null,{"className":"__className_b474bf footer_footerQuote__b7iXL","children":"Craft Beer Cares - Drinking beer for good Stuff since 2017"}]]}],["$","div",null,{"className":"footer_footerColumn__vJ4D7","children":["$","div",null,{"className":"footer_footerLogo__mj8G6","children":["$","$L3",null,{"src":"/assets/logo_bw.svg","alt":"Craft Beer Cares Logo","fill":true,"style":{"filter":"invert(1)"}}]}]}],["$","div",null,{"className":"footer_footerColumn__vJ4D7 footer_rightColumn__lXVIl","children":["$","div",null,{"className":"signUp_signUpBox__Wp2_Y","children":[["$","h4",null,{"className":"__className_b474bf signUp_title__On0QV","children":"Sign Up for Our Monthlyish Newsletters"}],["$","p",null,{"className":"__className_bc8ab8 signUp_signUpText__BCR9D","children":"Hear about upcoming events and initiatives from us and others pairing beer with charitable giving. We won’t share your info with anyone, and you can opt out at anytime."}],["$","input",null,{"className":"signUp_emailBox__O4KFn","placeholder":"Your e-mail"}],["$","button",null,{"className":"signUp_signUpButton__vx_P3","children":"Sign Up!"}]]}]}]]}]]}]}]}]]}]]}],null]]},["$","$L5",null,{"parallelRouterKey":"children","segmentPath":["children","about","children"],"loading":"$undefined","loadingStyles":"$undefined","loadingScripts":"$undefined","hasLoading":false,"error":"$undefined","errorStyles":"$undefined","errorScripts":"$undefined","template":["$","$L6",null,{}],"templateStyles":"$undefined","templateScripts":"$undefined","notFound":"$undefined","notFoundStyles":"$undefined","styles":[["$","link","0",{"rel":"stylesheet","href":"/_next/static/css/49201aecfc0e63ba.css","precedence":"next","crossOrigin":""}]]}]]},[null,["$","html",null,{"lang":"en","children":["$","body",null,{"children":["$","$L5",null,{"parallelRouterKey":"children","segmentPath":["children"],"loading":"$undefined","loadingStyles":"$undefined","loadingScripts":"$undefined","hasLoading":false,"error":"$undefined","errorStyles":"$undefined","errorScripts":"$undefined","template":["$","$L6",null,{}],"templateStyles":"$undefined","templateScripts":"$undefined","notFound":["$","div",null,{"className":"pageTemplate_pageContainer__cQnRZ","children":[["$","div",null,{"className":"pageTemplate_header__9KyCG","children":["$","div",null,{"className":"pageTemplate_content__jvw6J","children":["$","$L2",null,{}]}]}],["$","div",null,{"className":"pageTemplate_scrollContainer__0V5IP","children":[["$","main",null,{"className":"__className_bc8ab8 pageTemplate_content__jvw6J","children":["$","h1",null,{"children":"Page Not Found"}]}],["$","div",null,{"className":"pageTemplate_footer__SaaU7","children":["$","div",null,{"className":"pageTemplate_content__jvw6J","children":["$","footer",null,{"className":"$undefined","children":[["$","div",null,{"className":"footer_footerMobile__F33B9","children":["$","div",null,{"className":"footer_footerLogo__mj8G6","children":["$","$L3",null,{"src":"/assets/logo_bw.svg","alt":"Craft Beer Cares Logo","fill":true,"style":{"filter":"invert(1)"}}]}]}],["$","div",null,{"className":"footer_footerDesktop__h97Ga","children":[["$","div",null,{"className":"footer_footerColumn__vJ4D7","children":[["$","h4",null,{"className":"__className_b474bf footer_title__ua0Ts","children":"Craft Beer Cares"}],["$","nav",null,{"className":"__className_b474bf footer_footerLinks__khjBW","children":[["$","$L4","/",{"className":"footer_navLink__NX8Ib","href":"/","children":"Home"}],["$","$L4","/about",{"className":"footer_navLink__NX8Ib","href":"/about","children":"About"}],["$","$L4","/join_us",{"className":"footer_navLink__NX8Ib","href":"/join_us","children":"Join Us"}],["$","$L4","/events",{"className":"footer_navLink__NX8Ib","href":"/events","children":"Events"}],["$","$L4","/shop",{"className":"footer_navLink__NX8Ib","href":"/shop","children":"Shop"}],["$","$L4","/contact",{"className":"footer_navLink__NX8Ib","href":"/contact","children":"Contact"}]]}],["$","div",null,{"className":"footer_socialLinks__yVVge","children":[["$","a",null,{"href":"https://www.facebook.com/craftbeercares","className":"footer_socialLink__xJiBt","children":["$","$L3",null,{"src":"/assets/facebook.svg","alt":"Facebook","width":"30","height":"30"}]}],["$","a",null,{"href":"https://www.instagram.com/craftbeercares/","className":"footer_socialLink__xJiBt","children":["$","$L3",null,{"src":"/assets/instagram.svg","alt":"Instagram","width":"30","height":"30"}]}],["$","a",null,{"href":"https://twitter.com/craftbeercares","className":"footer_socialLink__xJiBt","children":["$","$L3",null,{"src":"/assets/twitter.svg","alt":"Twitter","width":"30","height":"30"}]}]]}],["$","div",null,{"className":"__className_b474bf footer_footerQuote__b7iXL","children":"Craft Beer Cares - Drinking beer for good Stuff since 2017"}]]}],["$","div",null,{"className":"footer_footerColumn__vJ4D7","children":["$","div",null,{"className":"footer_footerLogo__mj8G6","children":["$","$L3",null,{"src":"/assets/logo_bw.svg","alt":"Craft Beer Cares Logo","fill":true,"style":{"filter":"invert(1)"}}]}]}],["$","div",null,{"className":"footer_footerColumn__vJ4D7 footer_rightColumn__lXVIl","children":["$","div",null,{"className":"signUp_signUpBox__Wp2_Y","children":[["$","h4",null,{"className":"__className_b474bf signUp_title__On0QV","children":"Sign Up for Our Monthlyish Newsletters"}],["$","p",null,{"className":"__className_bc8ab8 signUp_signUpText__BCR9D","children":"Hear about upcoming events and initiatives from us and others pairing beer with charitable giving. We won’t share your info with anyone, and you can opt out at anytime."}],["$","input",null,{"className":"signUp_emailBox__O4KFn","placeholder":"Your e-mail"}],["$","button",null,{"className":"signUp_signUpButton__vx_P3","children":"Sign Up!"}]]}]}]]}]]}]}]}]]}]]}],"notFoundStyles":[["$","link","0",{"rel":"stylesheet","href":"/_next/static/css/760629fb8abc4698.css","precedence":"next","crossOrigin":""}]],"styles":null}]}]}],null]],[[["$","link","0",{"rel":"stylesheet","href":"/_next/static/css/1f969e4323e75552.css","precedence":"next","crossOrigin":""}]],"$L7"]]]]
7:[["$","meta","0",{"name":"viewport","content":"width=device-width, initial-scale=1"}],["$","meta","1",{"charSet":"utf-8"}],["$","title","2",{"children":"About"}],["$","link","3",{"rel":"icon","href":"/favicon.ico","type":"image/x-icon","sizes":"48x48"}],["$","meta","4",{"name":"next-size-adjust"}]]
1:null
