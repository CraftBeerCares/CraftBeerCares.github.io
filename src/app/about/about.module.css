.thanks {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
}

/* Tablet Smaller Desktop */
@media (min-width: 701px) {
  .thanks {
    flex-direction: row;
    justify-content: center;
  }
}
