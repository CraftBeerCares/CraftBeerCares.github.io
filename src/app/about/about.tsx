import { PageTemplate, ArticleHeader, ArticleRow, Thanks } from "@/components";
import { secondary } from "@/styles/fonts";
import styles from "./about.module.css";
import clsx from "clsx";

export const About = () => {
  return (
    <PageTemplate>
      <ArticleHeader imageSrc="/assets/003.webp" imageAlt="A picture of me">
        <h1 className={secondary.className}>The Craft Beer Cares Story</h1>
        <p>
          Craft Beer Cares was founded by Gautam Bhatnagar and Michaela
          Zelenanska in 2017. Inspired by other initiatives such as Cook for
          Syria that were happening at the time, the two took their social
          interests and contacts in craft beer and worked towards organising
          something that could raise money for great causes.
        </p>
        <p>
          Our first festival took place at BrewClub in Clapton, 1-2 July 2017.
          Since then, thanks to many friends and contacts within the craft beer
          industry as well as other wonderful friends, family members,
          volunteers, and enthusiastic attendees, we have held a grand total of
          4 beer festivals within 4 years, and have raised over £50,000 for
          charitable organisations.
        </p>
      </ArticleHeader>
      <ArticleRow
        imageSrc="/assets/001.webp"
        imageAlt="alt"
        imagePosition="right"
      >
        <p>
          Our most recent festival, Love Beer London, took place in partnership
          with the London Brewers Alliance and SIBA Southeast, and took place
          13-15 February 2020, only a short month before COVID-19 brought the UK
          into its first lockdown.
        </p>
        <p>
          Together, we managed to raise nearly £10,000 for The Benevolent, now
          called The Drinks Trust, a charity that provides support, care and
          assistance to people working in the drinks trade. Considering the
          upheaval many hospitality and drinks industry workers would find
          themselves in during the cycles of lockdown, furlough, reopening, and
          in some cases losing their jobs, we are glad to have been able to have
          contributed to such an incredible charity when we were able to do so.
        </p>
      </ArticleRow>
      <ArticleRow imageSrc="/assets/001.webp" imageAlt="alt">
        <p>
          Although our current plans for our next festival event may be some
          months off, if not perhaps even a bit longer, we at Craft Beer Cares
          are keen to promote other events, organisations, fund-raisers, and
          programmes that are supporting charities and non-profit organisations
          big and small, taking place throughout the UK.
        </p>
        <p>
          If you&apos;d like to receive news about charity events within the
          beer industry, please sign up for our newsletter. And if you have an
          event or an initiative you&apos;re holding for a charity or another
          great cause, please let us know by contacting us.
        </p>
      </ArticleRow>
      <div className={styles.thanks}>
        <Thanks />
      </div>
    </PageTemplate>
  );
};
