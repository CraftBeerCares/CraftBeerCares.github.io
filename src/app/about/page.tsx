import Footer from '../../components/footer'
import Header from '../../components/header'

export default function Page() {
  return (
  <div className="container">
    <Header />
    <main>About</main>
    <Footer />
  </div>
  )
}