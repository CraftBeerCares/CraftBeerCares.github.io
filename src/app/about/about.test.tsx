import { render } from "@testing-library/react";
import { About } from "./about";

it("renders About page", () => {
  const { container } = render(<About />);
  expect(container).toMatchSnapshot();
});
