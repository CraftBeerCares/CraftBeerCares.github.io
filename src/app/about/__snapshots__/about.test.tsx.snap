// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About page 1`] = `
<div>
  <div
    class="pageContainer"
  >
    <div
      class="header"
    >
      <header>
        <div
          class="header"
        >
          <button
            class="burgerMenuButton"
          >
            <img
              alt="Burger Menu"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              src="/assets/menu.svg"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent; filter: invert(1);"
            />
          </button>
          <h1
            class="className title"
          >
            Craft Beer Cares
          </h1>
          <nav
            class="topNav className"
          >
            <a
              class="navLink"
              href="/"
            >
              Home
            </a>
            <a
              class="navLink"
              href="/about"
            >
              About
            </a>
            <a
              class="navLink"
              href="/join_us"
            >
              Join Us
            </a>
            <a
              class="navLink"
              href="/events"
            >
              Events
            </a>
            <a
              class="navLink"
              href="/shop"
            >
              Shop
            </a>
            <a
              class="navLink"
              href="/contact"
            >
              Contact
            </a>
          </nav>
        </div>
      </header>
    </div>
    <div
      class="contentContainer"
    >
      <main
        class="className mainContent"
      >
        <div
          class="articleHeader"
        >
          <div
            class="imageWrapper"
          >
            <img
              alt="A picture of me"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fassets%2F003.webp&w=3840&q=75"
              srcset="/_next/image?url=%2Fassets%2F003.webp&w=640&q=75 640w, /_next/image?url=%2Fassets%2F003.webp&w=750&q=75 750w, /_next/image?url=%2Fassets%2F003.webp&w=828&q=75 828w, /_next/image?url=%2Fassets%2F003.webp&w=1080&q=75 1080w, /_next/image?url=%2Fassets%2F003.webp&w=1200&q=75 1200w, /_next/image?url=%2Fassets%2F003.webp&w=1920&q=75 1920w, /_next/image?url=%2Fassets%2F003.webp&w=2048&q=75 2048w, /_next/image?url=%2Fassets%2F003.webp&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: cover; color: transparent;"
            />
          </div>
          <article
            class="article articleLeft"
          >
            <h1
              class="className"
            >
              The Craft Beer Cares Story
            </h1>
            <p>
              Craft Beer Cares was founded by Gautam Bhatnagar and Michaela Zelenanska in 2017. Inspired by other initiatives such as Cook for Syria that were happening at the time, the two took their social interests and contacts in craft beer and worked towards organising something that could raise money for great causes.
            </p>
            <p>
              Our first festival took place at BrewClub in Clapton, 1-2 July 2017. Since then, thanks to many friends and contacts within the craft beer industry as well as other wonderful friends, family members, volunteers, and enthusiastic attendees, we have held a grand total of 4 beer festivals within 4 years, and have raised over £50,000 for charitable organisations.
            </p>
          </article>
        </div>
        <div
          class="articleRow"
        >
          <article
            class="article"
          >
            <p>
              Our most recent festival, Love Beer London, took place in partnership with the London Brewers Alliance and SIBA Southeast, and took place 13-15 February 2020, only a short month before COVID-19 brought the UK into its first lockdown.
            </p>
            <p>
              Together, we managed to raise nearly £10,000 for The Benevolent, now called The Drinks Trust, a charity that provides support, care and assistance to people working in the drinks trade. Considering the upheaval many hospitality and drinks industry workers would find themselves in during the cycles of lockdown, furlough, reopening, and in some cases losing their jobs, we are glad to have been able to have contributed to such an incredible charity when we were able to do so.
            </p>
          </article>
          <div
            class="articleImage"
          >
            <img
              alt="alt"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fassets%2F001.webp&w=3840&q=75"
              srcset="/_next/image?url=%2Fassets%2F001.webp&w=640&q=75 640w, /_next/image?url=%2Fassets%2F001.webp&w=750&q=75 750w, /_next/image?url=%2Fassets%2F001.webp&w=828&q=75 828w, /_next/image?url=%2Fassets%2F001.webp&w=1080&q=75 1080w, /_next/image?url=%2Fassets%2F001.webp&w=1200&q=75 1200w, /_next/image?url=%2Fassets%2F001.webp&w=1920&q=75 1920w, /_next/image?url=%2Fassets%2F001.webp&w=2048&q=75 2048w, /_next/image?url=%2Fassets%2F001.webp&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent;"
            />
          </div>
        </div>
        <div
          class="articleRow"
        >
          <div
            class="articleImage"
          >
            <img
              alt="alt"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fassets%2F001.webp&w=3840&q=75"
              srcset="/_next/image?url=%2Fassets%2F001.webp&w=640&q=75 640w, /_next/image?url=%2Fassets%2F001.webp&w=750&q=75 750w, /_next/image?url=%2Fassets%2F001.webp&w=828&q=75 828w, /_next/image?url=%2Fassets%2F001.webp&w=1080&q=75 1080w, /_next/image?url=%2Fassets%2F001.webp&w=1200&q=75 1200w, /_next/image?url=%2Fassets%2F001.webp&w=1920&q=75 1920w, /_next/image?url=%2Fassets%2F001.webp&w=2048&q=75 2048w, /_next/image?url=%2Fassets%2F001.webp&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent;"
            />
          </div>
          <article
            class="article"
          >
            <p>
              Although our current plans for our next festival event may be some months off, if not perhaps even a bit longer, we at Craft Beer Cares are keen to promote other events, organisations, fund-raisers, and programmes that are supporting charities and non-profit organisations big and small, taking place throughout the UK.
            </p>
            <p>
              If you'd like to receive news about charity events within the beer industry, please sign up for our newsletter. And if you have an event or an initiative you're holding for a charity or another great cause, please let us know by contacting us.
            </p>
          </article>
        </div>
        <div
          class="thanks"
        >
          <div
            class="quoteBox"
          >
            <img
              alt="Quote Start"
              class="quoteImage"
              data-nimg="1"
              decoding="async"
              height="24"
              loading="lazy"
              src="/assets/quote_start.svg"
              style="color: transparent;"
              width="24"
            />
            <p
              class="quote"
            >
              The generous support from Craft Beer Cares is huge for Art Against Knives.
            </p>
            <img
              alt="Quote End"
              class="quoteImage"
              data-nimg="1"
              decoding="async"
              height="24"
              loading="lazy"
              src="/assets/quote_end.svg"
              style="color: transparent;"
              width="24"
            />
            <p
              class="author"
            >
              Katy Dawe, Art Against Knives Director
            </p>
          </div>
          <div
            class="quoteBox"
          >
            <img
              alt="Quote Start"
              class="quoteImage"
              data-nimg="1"
              decoding="async"
              height="24"
              loading="lazy"
              src="/assets/quote_start.svg"
              style="color: transparent;"
              width="24"
            />
            <p
              class="quote"
            >
              We'd like to say a huge thank you to Craft Beer Cares for choosing to support Mind through their first beer festival. Every penny raised will fund Mind's vital work.
            </p>
            <img
              alt="Quote End"
              class="quoteImage"
              data-nimg="1"
              decoding="async"
              height="24"
              loading="lazy"
              src="/assets/quote_end.svg"
              style="color: transparent;"
              width="24"
            />
            <p
              class="author"
            >
              Karen Bolton, Community Fundraising Manager for Mind
            </p>
          </div>
          <div
            class="quoteBox"
          >
            <img
              alt="Quote Start"
              class="quoteImage"
              data-nimg="1"
              decoding="async"
              height="24"
              loading="lazy"
              src="/assets/quote_start.svg"
              style="color: transparent;"
              width="24"
            />
            <p
              class="quote"
            >
              From November through to March, the coldest months of the year, Hackney Winter Night Shelter opens its doors to the most vulnerable in our community. Any money raised would be of enormous benefit.
            </p>
            <img
              alt="Quote End"
              class="quoteImage"
              data-nimg="1"
              decoding="async"
              height="24"
              loading="lazy"
              src="/assets/quote_end.svg"
              style="color: transparent;"
              width="24"
            />
            <p
              class="author"
            >
              Christina Ball, Hackney Winter Night Shelter
            </p>
          </div>
        </div>
      </main>
      <footer
        class="footer"
      >
        <div
          class="footerMobile"
        >
          <div
            class="footerLogo"
          >
            <img
              alt="Craft Beer Cares Logo"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              src="/assets/logo_bw.svg"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; filter: invert(1);"
            />
          </div>
        </div>
        <div
          class="footerDesktop"
        >
          <div
            class="footerColumn"
          >
            <h4
              class="className title"
            >
              Craft Beer Cares
            </h4>
            <nav
              class="className footerLinks"
            >
              <a
                class="navLink"
                href="/"
              >
                Home
              </a>
              <a
                class="navLink"
                href="/about"
              >
                About
              </a>
              <a
                class="navLink"
                href="/join_us"
              >
                Join Us
              </a>
              <a
                class="navLink"
                href="/events"
              >
                Events
              </a>
              <a
                class="navLink"
                href="/shop"
              >
                Shop
              </a>
              <a
                class="navLink"
                href="/contact"
              >
                Contact
              </a>
            </nav>
            <div
              class="socialLinks"
            >
              <a
                class="socialLink"
                href="https://www.facebook.com/craftbeercares"
              >
                <img
                  alt="Facebook"
                  data-nimg="1"
                  decoding="async"
                  height="30"
                  loading="lazy"
                  src="/assets/facebook.svg"
                  style="color: transparent;"
                  width="30"
                />
              </a>
              <a
                class="socialLink"
                href="https://www.instagram.com/craftbeercares/"
              >
                <img
                  alt="Instagram"
                  data-nimg="1"
                  decoding="async"
                  height="30"
                  loading="lazy"
                  src="/assets/instagram.svg"
                  style="color: transparent;"
                  width="30"
                />
              </a>
              <a
                class="socialLink"
                href="https://twitter.com/craftbeercares"
              >
                <img
                  alt="Twitter"
                  data-nimg="1"
                  decoding="async"
                  height="30"
                  loading="lazy"
                  src="/assets/twitter.svg"
                  style="color: transparent;"
                  width="30"
                />
              </a>
            </div>
            <div
              class="className footerQuote"
            >
              Craft Beer Cares - Drinking beer for good Stuff since 2017
            </div>
          </div>
          <div
            class="footerColumn"
          >
            <div
              class="footerLogo"
            >
              <img
                alt="Craft Beer Cares Logo"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                src="/assets/logo_bw.svg"
                style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; filter: invert(1);"
              />
            </div>
          </div>
          <div
            class="footerColumn rightColumn"
          >
            <div
              class="signUpBox"
            >
              <h4
                class="className title"
              >
                Sign Up for Our Monthlyish Newsletters
              </h4>
              <p
                class="className signUpText"
              >
                Hear about upcoming events and initiatives from us and others pairing beer with charitable giving. We won’t share your info with anyone, and you can opt out at anytime.
              </p>
              <input
                class="emailBox"
                placeholder="Your e-mail"
              />
              <button
                class="signUpButton"
              >
                Sign Up!
              </button>
            </div>
          </div>
        </div>
      </footer>
    </div>
  </div>
</div>
`;
